vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -30,6 +30,63 @@ module InlineParameterNameHints =
     let private doesFieldNameExist (field: FSharpField) = 
         not field.IsNameGenerated
 
+    // copypaste: https://stackoverflow.com/a/15993446/3232646
+    let private allIndexesOf (value: string) (s: string)  =
+        seq {
+            let mutable minIndex = s.IndexOf(value);
+            while (minIndex <> -1) do
+                yield minIndex;
+                minIndex <- s.IndexOf(value, minIndex + value.Length);
+        }
+
+    // Fragile Roslyn arithmetics, don't try this at home.
+    // 
+    // Why the hell is this so complicated? 
+    // There can be (rarely) cases like there can be cases like 
+    // <somecode>.SymbolUse1().SymbolUse2()<morecode>
+    // and we need to locate the last one.
+    // Hopefully someone will have a better way to do that one day,
+    // this is quite heavily tested so should be safe to refactor.
+    let getSymbolPosition
+        (symbolUse: FSharpSymbolUse) 
+        (source: Microsoft.CodeAnalysis.Text.SourceText) = 
+
+        let symbolName = symbolUse.Symbol.DisplayNameCore
+        let symbolLine = symbolUse.Range.End.Line - 1
+        let symbolRow = source.Lines.[symbolLine]
+        let symbolColumns = $"{symbolRow}" |> allIndexesOf symbolName
+
+        if symbolColumns |> Seq.isEmpty
+        then None
+        else 
+            let symbolColumn = 
+                symbolColumns
+                |> Seq.where (fun column -> column < symbolUse.Range.EndColumn)
+                |> Seq.last
+
+            let positionLine = symbolLine + 1
+            let positionColumn = symbolColumn + symbolName.Length + 1
```

Comment:
I don't get it - why isn't this always the same as `symbolUse.Range.EndColumn`? Is the range incorrect if there are multiple occurrences of the same symbol on one line?

Reply:
Hold my beer

Reply:
Okay all this turned out to be total crap, thanks for asking :D 

I pushed that stuff before I fixed the issue in the test framework about the symbol loading, it wasn't working otherwise. But then I dug into the symbol problem and forgot to check if this stuff here is still relevant - it's not :shrug: